openshift/e2e-common.sh

```diff
@@ -226,16 +259,16 @@ function run_e2e_tests(){
   # Prevent HPA from scaling to make the tests more stable
   oc -n "$SERVING_NAMESPACE" patch hpa activator \
   --type 'merge' \
-  --patch '{"spec": {"maxReplicas": '2', "minReplicas": '2'}}' || return 1
-
-  # Give the controller time to sync with the rest of the system components.
-  sleep 30
+  --patch '{"spec": {"maxReplicas": '${REPLICAS}', "minReplicas": '${REPLICAS}'}}' || return 1
 
   # Use sed as the -spoofinterval parameter is not available yet
   sed "s/\(.*requestInterval =\).*/\1 10 * time.Millisecond/" -i vendor/knative.dev/pkg/test/spoof/spoof.go
 
+  # Run HA tests separately as they're stopping core Knative Serving pods
+  # Define short -spoofinterval to ensure frequent probing while stopping pods
   go_test_e2e -tags=e2e -timeout=15m -failfast -parallel=1 \
     ./test/ha \
+    -replicas="${REPLICAS:-1}" -buckets="${BUCKETS:-1}" -spoofinterval="10ms" \
```

Comment:
I'd say it's not necessary to put the `:-1` as a default value. We're defining the vars just a few lines above and we don't use this approach even for the `--patch`. If the value is not set it will fail there already.

Reply:
Thank you. Updated.